language/testing-infra/e2e-tests/goldens/language_e2e_testsuite__tests__transaction_builder__test_rotate_authentication_key_with_nonce_admin_version_2.exp

```diff
@@ -1,2 +1,2 @@
+Ok([TransactionOutput { write_set: WriteSet(WriteSetMut { write_set: [(AccessPath { address: 0000000000000000000000000a550c18, path: 01000000000000000000000000000000010b4469656d4163636f756e740b4469656d4163636f756e7400 }, Value(201304972f9242cbc3528a1e286323471ab891baa37e0053b85651693a79854a00010000000000000000000000000a550c18010000000000000000000000000a550c1800000000000000001802000000000000000000000000000000000000000a550c1800000000000000001803000000000000000000000000000000000000000a550c180400000000000000)), (AccessPath { address: 0000000000000000000000000a550c18, path: 01000000000000000000000000000000010b4469656d4163636f756e74134469656d57726974655365744d616e6167657200 }, Value(01000000000000001801000000000000000000000000000000000000000a550c18)), (AccessPath { address: 3eebd722df1633f4bf38f1e2087222e9, path: 01000000000000000000000000000000010b4469656d4163636f756e740b4469656d4163636f756e7400 }, Value(207deeccb1080854f499ec8b4c1b213b82c5e34b925cf6875fec02d4b77adbd2d6013eebd722df1633f4bf38f1e2087222e9013eebd722df1633f4bf38f1e2087222e900000000000000001800000000000000003eebd722df1633f4bf38f1e2087222e900000000000000001800000000000000003eebd722df1633f4bf38f1e2087222e90000000000000000))] }), events: [ContractEvent { key: EventKey([1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10, 85, 12, 24]), index: 0, type: Struct(StructTag { address: 00000000000000000000000000000001, module: Identifier("DiemAccount"), name: Identifier("AdminTransactionEvent"), type_params: [] }), event_data: "0000000000000000" }], gas_used: 0, status: Keep(EXECUTED) }])
```

Comment:
This had to change when after cherry picking, and not really sure why. Any ideas, @bob-wilson @tzakian @runtian-zhou @vgao1996?

Reply:
WAG #8288?

Reply:
I'm not sure about the "1" -> "2" change in one of the values, but the "index" value that changed from "1" -> "0" suggests that maybe there was an empty ChangeSet (and maybe the "2" is the count of the ChangeSets??). I'm really guessing here, though